spec/expression.dd

```diff
@@ -1144,6 +1204,55 @@ $(H4 $(LNAME2 cast_class, Class References))
         (i.e. a reinterpret cast).
     )
 
+$(H4 $(LNAME2 cast_pointers, Pointers))
+    $(P Casting a pointer modifies the type that will be obtained as a result of dereferencing,
+        along with the number of bytes on which pointer arithmetic is performed.
+    )
```

Comment:
Please be concise. You are talking about casting a pointer type to another pointer type. From your wording that is not obvious, sincer you can cast a pointer to another pointer or to a basic type, or to a class/struct (and the other way around).